app/views/jst/courses/roster/EditEnrollmentsView.handlebars

```diff
@@ -0,0 +1,15 @@
+<p>{{#t "edit_sections_desc"}}Please select a module to start this student at.{{/t}}</p>
```

Comment:
This translation is currently breaking the test suite in LMS.  I'm not sure how translations work in JS in Canvas.  Need to add missing translation key or not translate this text